### ISSUE TITLE
Lip is so small and can not handle long range face

### ISSUE THREAD

**Issue by yuanjunchai** [Weizhi-Zhong/IP_LAP#29]
Hi Dear,
Your work is so excellent! But after test many videos, I find generated lip is so small and it can not handle long range face. Could you have any ideas to solve these issues?

**Comment by KingStorm** [comment]
I guess the small lip is related to head motion (i.e., your head motion is nearly still)

what do you mean about the long range face?

**Comment by yuanjunchai** [comment]
Hi Jianhao!

First, I do not understand the relationship between head motion and lip
openness. You mean the more head motion, the better lip open size?
Second, the long-range is compared to the low-range face, which means the
small face, captured by long range lens, appears in a high-resolution video.

Jianhao Ye ***@***.***> 于2023年8月5日周六 22:44写道：

**Comment by KingStorm** [comment]
> 

Yeah I was to mean there are entanglement between head motion and lip syncing, It depends on your training dataset. If in your training dataset, the speaker only speaks when head moves. This would make the model to learn that relationship, i.e., speak when head motion moves -> lip is small when head motion is still.

BTW, what do you observer in your long-range face case?

**Comment by sunjian2015** [comment]
> lip is small when head motion is still

I have encountered this problem, how should i solve it?
